lib/spack/docs/getting_started.rst

```diff
@@ -202,6 +202,16 @@ installed, but you know that new compilers have been added to your
 This loads the environment module for gcc-4.9.0 to add it to
 ``PATH``, and then it adds the compiler to Spack.
 
+.. note::
+
+   After running ``spack compiler find``, if spack identifies a new
```

Comment:
@mwkrentel 

I like the message but think it could be more succinct/direct:

> When Spack adds a compiler with ``spack compiler find``, it does not register modules needed for the compiler. If the compiler requires modules, you must edit `compilers.yaml` manually to add the module: see :ref:`compilers-requiring-modules` for more details.

or even more direct:

> If a compiler requires modules, then you must manually add it to `compilers.yaml` for Spack to use it: see :ref:`compilers-requiring-modules` for more details.

My goal is to balance useful information with keeping the length of the "Getting Started" section manageable.